### PR TITLE
Adding release instruction to verify quickstart-scala template

### PR DESCRIPTION
**Pull request by leo-da**
based on quickstart-scala/README.md
Related to: #2776

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: release/RELEASE.md [hunk 1]
```diff
@@ -56,6 +56,19 @@
    1. Right click on `eurBank` in line 17 and verify that goto
       definition takes you to the definition in line 14.
    1. Close all files.
+   1. Create a new project with `daml new ./quickstart-scala quickstart-scala`
+      and switch to it using `cd ./quickstart-scala`
+   1. Run `daml build`
+   1. Run `daml codegen scala`
+   1. In 2 separate terminals (since each command will block) run
```
Comment (S11001001) [inline release/RELEASE.md]:
```suggestion
   1. In 2 separate terminals (since the first command will block) run
```

**Comment by leo-da** [comment]
The release is already pretty expensive, a lot of manual steps. Adding 3 additional steps going to make the verification steps 10 min longer, that is for sure. The daml assistant integration test is in TODO, too much stuff happening in the run-time team. Would appreciate any help with the test though :)

**Comment by cocreature** [comment]
It currently takes me around 15 minutes to test a release, adding 10 minutes to that is just not reasonable imho. I realize that your busy and don’t want to prioritize tests right now but slowing down the release process for everyone instead of spending some time on testing is not a good tradeoff.

**Comment by gerolf-da** [comment]
Adding more and more manual steps indeed doesn't scale well. How about we find a way to automate these kind of tests?

**Comment by gerolf-da** [comment]
@leo-da: please have another look at getting #1711 over the finish line.

**Pull request closed without merging**